app/views/layouts/result_content.erb

```diff
@@ -58,11 +58,40 @@
         </div>
       </div>
     </div>
+    <!-- Progress Bar -->
     <section id='content'>
+      <div class="progress_meter">
+        <span id="progress_span"><p>
```

Comment:
I'm not convinced the `p` tag is necessary (seems like we can just use the span). If you can remove this, that would be great (and from the css as well).